doc/mail.md

```diff
@@ -4,6 +4,7 @@ The `@nixos.org` email server is currently hosted on [ImprovMX](https://improvmx
 
 ImprovMX is configured to forward emails on the following addresses to the respective users:
 - community@nixos.org: moderation@nixos.org
+- sponsor@nixos.org: steering@nixos.org, foundation@nixos.org
```

Comment:
Is the ping for the alias?

Reply:
Yeah, the infra team should implement this change, but only once merged.

Reply:
cc @jfly

Reply:
In addition to the SC approval, this PR now also needs an approval from the infra team though :)

Reply:
I'm not sure an approval is necessary but they should implement it?

Reply:
See https://github.com/NixOS/infra/pull/608, and the caveat in the PR message.

Reply:
Approval is necessary because infra is a code owner of this file, and we require approvals from code owners before merging.